### DISCUSSION TITLE
Exception when creating context with UsernamePasswordAuthenticationProvider (MS-Trust)

### DISCUSSION THREAD

**Discussion by vroesgen** [pnp/pnpcore#823] [Q&A]
Since last week I suddenly get an exception when creating context with UsernamePasswordAuthenticationProvider .
Everything was working since 6 months.

Exception message:
There was an error parsing WS-Trust response from the endpoint. This may occur if there is an issue with your ADFS configuration

InnerException.Message:
Federated service at https://autologon.microsoftazuread-sso.com/biofrontera.com/winauth/trust/2005/usernamemixed?client-request-id=5ee9acfa-c7ff-4221-abf7-dc524c819bf2 returned error: Authentication Failure

I found in this document [https://support.dotdigital.com/hc/en-gb/articles/4903754728594--April-2022-Microsoft-authentication-update](https://github.com/pnp/pnpcore/issues/url) that WS-Trust protocol has been removed by Microsoft sind April 1, 2022

Does anyone have an idea where the cause could be?

**Comment by jansenbe** [comment]
Following up via https://github.com/pnp/pnpcore/issues/822
